controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiHandler.java

```diff
@@ -1814,6 +1814,12 @@ private void toSlime(Cursor response, DeploymentId deploymentId, Deployment depl
 
         application.projectId().ifPresent(i -> response.setString("screwdriverId", String.valueOf(i)));
 
+        controller.applications().decideCloudAccountOf(deploymentId, application.deploymentSpec()).ifPresent(cloudAccount -> {
+            Cursor enclave = response.setObject("enclave");
+            enclave.setString("cloudAccount", cloudAccount.value());
+            enclave.setString("athensDomain", controller.zoneRegistry().cloudAccountAthenzDomain(cloudAccount).value());
```

Comment:
Call it `athenzDomain`?

Reply:
I've considered that, but then I saw that `/application/v4` already used "athensDomain":
https://github.com/vespa-engine/vespa/blob/abe8f597b50d5fcfb379c136abc6462044f90e0c/controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiHandler.java#L2527
https://github.com/vespa-engine/vespa/blob/abe8f597b50d5fcfb379c136abc6462044f90e0c/controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiHandler.java#L2664